clients/extension/scripts/popup.js

```diff
@@ -120,7 +124,7 @@ function getSiteUrl() {
 
       const url = new URL(tabs[0].url);
       if (url.hostname === 'localhost') {
-        reject(new Error('Use DevTools to audit pages on localhost.'));
+        reject(new Error(STRINGS.localhostErrorMessage));
       } else if (/^(chrome|about)/.test(url.protocol)) {
```

Comment:
Any firefox specific urls we need to include here now? `firefox://page`?

Reply:
just abouts i think

about:about
![image](https://user-images.githubusercontent.com/4071474/74565203-93c88e00-4f25-11ea-893e-170d3715836d.png)